separate PQ list and bookmark list (fix #11458)

moving-bits opened the pull request:
## Description
Separates PQ list and bookmark list to avoid overcrowding.
"Show all" toggles between downloadable/all for PQ list, and between new/all for bookmark lists.

fm-sys commented on main/res/values/strings.xml:
```diff
@@ -345,7 +345,8 @@
     <string name="menu_scan_geo">Scan geocode</string>
-    <string name="menu_lists_pocket_queries">Lists / pocket queries</string>
+    <string name="menu_lists_pocket_queries">Pocket queries</string>
+    <string name="menu_lists_bookmarklists">Bookmarklists</string>
```
IMHO, there should be a space between "bookmark" and "lists" in English

moving-bits replied:
Or maybe call it just `Bookmarks`?

fm-sys replied:
I just have browsed a bit through their website and app, and it seems there's not a single place where they use the word "bookmark" anymore.

We still need "bookmark" to distinguish between our offline lists, but at least this means we cannot strip the word "lists" 

![Screenshot_20210912_154235_com groundspeak geocaching intro_copy_432x1028](https://user-images.githubusercontent.com/64581222/132990028-918e8788-0c52-435d-9952-9af435e1709f.jpg)

moving-bits replied:
Ok, changed it to bookmark lists

fm-sys reviewed (commented):
Looks good in general. Maybe we should rename our `PocketQuery.java` class to something different as well (as it is used for bookmark lists as well as for PQs)

moving-bits commented:
> Maybe we should rename our `PocketQuery.java` class to something different as well (as it is used for bookmark lists as well as for PQs)

We could rename it to `Bookmarklist.java` or `Bookmarks.java` (or something similar), as pocket queries are sort of a subclass of bookmark lists.

fm-sys commented:
> > Maybe we should rename our `PocketQuery.java` class to something different as well (as it is used for bookmark lists as well as for PQs)
> 
> We could rename it to `Bookmarklist.java` or `Bookmarks.java` (or something similar), as pocket queries are sort of a subclass of bookmark lists.

Or maybe `GCLists`?

fm-sys commented:
> > We could rename it to `Bookmarklist.java` or `Bookmarks.java` (or something similar), as pocket queries are sort of a subclass of bookmark lists.
> 
> Or maybe `GCLists`?

Or maybe more general `OnlineList`...

moving-bits commented:
> > Or maybe `GCLists`?
> 
> Or maybe more general `OnlineList`...

I've refactored it to `GCList` as it is still GC-specific. Renamed the related layout files as well.
